api/envoy/data/accesslog/v3/accesslog.proto

```diff
@@ -266,6 +266,9 @@ message ResponseFlags {
 
   // Indicates there was a max stream duration reached on the upstream request.
   bool upstream_max_stream_duration_reached = 20;
+
+  // Indicates the response was served from a sidecar cache.
```

Comment:
I assume "sidecar cache" is meant to include more than just CacheFilter caches, but I think the term is too ambiguous. How about "from a cache filter", or just "from CacheFilter" (and update it when some other Envoy cache comes along).

Reply:
Yeah, I think for that case the best option today (well soon) would be to set the new "don't override date header" config knob, the PR which prompted this whole discussion

Reply:
Ack.  think it's worth renaing from from_cache -> from_cache_filter everywhere?

Reply:
Yes, the extra clarity seems worth it.